hasura/metadata/databases/default/tables/public_app_metadata.yaml

```diff
@@ -196,13 +202,7 @@ update_permissions:
                 users:
                   id:
                     _eq: X-Hasura-User-Id
-      filter: {}
 delete_permissions:
-  - role: reviewer
```

Comment:
FYI, it is possible that an app ends up without any app_metadata, code should account for that

Reply:
yeah we account for this in the update and fetch